src/components/mixins/DownloadMixin.vue

```diff
@@ -85,7 +85,8 @@ export default class DownloadMixin extends Vue {
         assignId: number
     ): Promise<void> {
         await ProfileModList.requestLock(async () => {
-            const modList: ManifestV2[] = await installModsToProfile(downloadedMods, profile, undefined,(status, modName, installProgress) => {
+            const reloadModList = () => this.$store.dispatch('profile/tryLoadModListFromDisk');
```

Comment:
This could use `updateModList` instead of `tryLoadModListFromDisk`, but that would then require passing the mod list from `ProfileUtils`' `installModsToProfile`.
I thought this way would be better as the thing was to make the list match what's on the disk to begin with.